src/android/CameraLauncher.java

```diff
@@ -994,6 +994,7 @@ private Bitmap getScaledAndRotatedBitmap(String imageUrl) throws IOException {
                         // read exifData of source
                         exifData = new ExifHelper();
                         exifData.createInFile(filePath);
```

Comment:
OT to your PR, but man is this a bad variable file. I read it as "create in file", but it actually means "create in(put)file".